usagov/usagov-2021
# USAGOV-1328 analytics dashboard seection of dropdown items causes page refresh

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-

## Description

Added a submit button to the dropdown for page select

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps

- This one is a little weird to test. First see the current behavior on usa.gov/website-analytics. 
- You won't use localhost to test this but when you get the repo for the PR if using VSCode you can use the Live Server extension to open a live preview.
  - Go to website-analytics/index.html in VSCode and click Live Server from that file 
- When it runs you should see the page with no data, that's fine (see below). 
- Click different selections in the dropdown menu. Previously clicking the selection automatically took you to that page. Now it should wait until you press "Go". 
- Make sure the selections work on each of the three pages. 


You don't need to for this test but if you want to: In the preview you'll see the data isn't pulling from the database. 
- For each "data-source={url }" 
- Go to the link in the datasource and download that data to a json file locally. 
- Replace all the links with references to your local files. 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
